dashboard/views.py

```diff
@@ -460,7 +460,10 @@ def grade_distribution(request, course_id=0):
             'course_id': course_id,
             'enrollment_type': 'StudentEnrollment'
         })
-    if df.empty or df.count().current_grade < 6:
+    if len(df) <= config.GRADE_DISTRIBUTION_MINIMUM:
+        logger.error(f"Course enrollment count {len(df)} is lower than the minimum required to display the Grade Distribution visualization {config.GRADE_DISTRIBUTION_MINIMUM}")
+        return HttpResponse(json.dumps({'gd_disable':'true','gd_msg': f'Grade Distribution view is disabled because the course enrollment is less than {config.GRADE_DISTRIBUTION_MINIMUM}'}), content_type='application/json')
+    if df.empty or df.count().current_grade < 6 :
```

Comment:
is the check for 6 still necessary, given the setting of `config.GRADE_DISTRIBUTION_MINIMUM`?

If so, could we make the number 6 as another admin config variable?

Reply:
10 is the number we set as default. But some universities set it 0 (with out knowingly) then the Grade distribution need atleast 6 grades for rest of the depersonalized logic to work. So it need to check for GD to work. 
This is not something new I introduced with this change [it was there from before](https://github.com/tl-its-umich-edu/my-learning-analytics/blob/master/dashboard/views.py#L463). 
I don't feel this needs another configuration value.

Reply:
I see. 

Can we expose the number 6 to the configuration description text? Something like `Minimum number of student enrollments required to show the Grade Distribution visualization. The number needs to be greater than 6 for depersonalized logic to work in the Grade Distribution view. `. WDYT?

Reply:
Ok, I can do that.

Reply:
Done